docs/internals/contributing/writing-code/accessibility.txt

```diff
@@ -0,0 +1,123 @@
+=============
+Accessibility
+=============
+
+The Django project is committed to being accessible to everyone
+in all possible ways.
+We work to conform to established accessibility standards,
+and also to meet our own targets for supporting specific input devices
+and assistive technologies.
+
+.. _accessibility-standards:
+
+Accessibility standards
+=======================
+
+We work to conform with the `Web Content Accessibility Guidelines`_ (WCAG),
+version 2.2, at the AA level.
+WCAG is the most established standard for web accessibility.
+AA-level guidelines are the most common legal compliance target worldwide.
+
+We also attempt to follow other best practices, such as:
+
+* WCAG 2.2 AAA-level guidelines.
+* The upcoming `WCAG 3.0`_ guidelines.
+* `Authoring Tool Accessibility Guidelines`_ (ATAG) 2.0 – guidelines
+  for software and services that are used to produce web content.
+
+To learn more about accessibility without diving straight into standards,
+we recommend `The A11Y Project`_, a community-driven effort
+to make digital accessibility easier.
+
+.. _Web Content Accessibility Guidelines: https://www.w3.org/TR/WCAG22/
+.. _WCAG 3.0: https://www.w3.org/TR/wcag-3.0/
+.. _Authoring Tool Accessibility Guidelines: https://www.w3.org/TR/ATAG20/
+.. _The A11Y Project: https://www.a11yproject.com/
+
+.. _accessibility-support-targets:
+
+Accessibility support targets
```

Comment:
```suggestion
Support scope
```

Reply:
Could you explain your suggestion? I have spent a lot of time on wording choices here, and based this on other existing guidelines that use this wording.